guides/release/components/block-content.md

```diff
@@ -114,21 +131,26 @@ supplied by the `<Message>` tag.
 
 The way to do this in Ember is by using the `{{yield}}` syntax.
 
-```handlebars {data-filename="app/components/message.hbs"}
-<Message::Avatar
-  @title={{@avatarTitle}}
-  @initial={{@avatarInitial}}
-  @isActive={{@userIsActive}}
-  class="{{if @isCurrentUser "current-user"}}"
-/>
-<section>
-  <Message::Username
-    @name={{@username}}
-    @localTime={{@userLocalTime}}
+```gjs {data-filename="app/components/message.gjs"}
+import MessageAvatar from 'my-app/components/message/avatar';
```

Comment:
same here, relative imports.

These should also be using named exports/imports, since we don't need defaults anymore (and default exports are undeniably worse for userland code (default exports are only a requirement for frameworks to call "something" in the file)).

Named exports will keep the names consistent.

Additionally, we don't need to have the `Message` prefix on each of these components, since the context is already there since we are in the message component and message folder.

let's reduce the visual noise/verbosity.